config.yaml

```diff
@@ -42,34 +42,18 @@ build_iso:
     packages_dir: "result/packages/latest"
```

Comment:
You are deleting the previous OP Host OS virtualization-hypervisor package group...